cime_config/buildexe

```diff
@@ -8,7 +8,10 @@ import sys, os
 
 _CIMEROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..","..","..","..")
 if not os.path.basename(_CIMEROOT)=="cime":
-    _CIMEROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..","..","..","..","..","cime")
+    _CIMEROOT = os.path.abspath(os.path.join(os.path.dirname(os.path.abspath(__file__)), "..","..","..","..","..","cime"))
```

Comment:
@mvertens has introduced a new env variable CIMEROOT, we should be consistant and do that here as well.
Example: 
_CIMEROOT = os.environ.get("CIMEROOT")
if _CIMEROOT is None:
    raise SystemExit("ERROR: must set CIMEROOT environment variable")
sys.path.append(os.path.join(_CIMEROOT, "scripts", "Tools"))
sys.path.append(os.path.join(_CIMEROOT, "scripts", "Tools"))